posthog/models/event/query_event_list.py

```diff
@@ -33,10 +32,8 @@ def determine_event_conditions(
             result += "AND timestamp < %(before)s"
             params.update({"before": timestamp})
         elif k == "person_id":
-            result += """AND distinct_id IN (%(distinct_ids)s)"""
-            person = get_pk_or_uuid(Person.objects.all(), v).first()
-            distinct_ids = person.distinct_ids if person is not None else []
-            params.update({"distinct_ids": list(map(str, distinct_ids))})
+            result += "AND person_id = %(person_id)s"
```

Comment:
make this a conditional on `team.actor_on_events_querying_enabled` for now!

Reply:
Great thanks 🙏